manifests/repo/yum_postgresql_org.pp

```diff
@@ -1,5 +1,7 @@
 # PRIVATE CLASS: do not use directly
-class postgresql::repo::yum_postgresql_org inherits postgresql::repo {
+class postgresql::repo::yum_postgresql_org (
+      $repo_priority = 10
```

Comment:
This is a private class, and thus should have no parameters.

Either the priority should be hard-coded to some value, or made a parameter on postgresql::globals.